lib/struct.js

```diff
@@ -184,6 +186,16 @@ function toString () {
 }
 
 /**
+ * Change struct padding mode.
+ *
+ * @api public
+ */
+function usePack (flag) {
```

Comment:
yeah.... actually that is not need.

this is design problem, i think, `Struct`'s  second param have low readability
i like python way like this; `struct = Struct(spec, packed=True)`
btw, current javascript not support named param yet,

so sometimes i will use this instead that
```javascript
var struct = new Struct(spec);
struct.usePack(true);
```
but also, i'm not to change on runtime env

2nd idea, introduce new class like `PackedStruct` is more than good solution about this problem.

few month ago, i was only had first idea... :(